xds/src/main/java/com/linecorp/armeria/xds/BootstrapClusters.java

```diff
@@ -51,8 +54,13 @@ public void snapshotUpdated(ClusterSnapshot newSnapshot) {
     }
 
     @Nullable
-    ClusterSnapshot get(String name) {
-        return clusterSnapshots.get(name);
+    ClusterSnapshot clusterSnapshot(String clusterName) {
```

Comment:
Question) Should `ClusterRoot` also refer to this preassigned `clusterName` for consistency?

Reply:
Oops, let me fix that.